### DISCUSSION TITLE
conditional formatting with spans 1.3 release vs 1.4 pre-release

### DISCUSSION THREAD

**Discussion by anielsen001** [quarto-dev/quarto-cli#8064] [Q&A] [answered]
### Description

I'm trying to follow the documentation [example for conditional formatting](https://quarto.org/docs/authoring/conditional.html) using spans. When I do this with version 1.3.450, the information in the square brackets never appears in either pdf or html format It just says `Some text .` . When I use the pre-release 1.4.533 I get the expected conditional behavior. 
 
Is this a pre-release only feature? the documentation mentions some pre-release only behavior later, but not about this topic.

My implementation of the example follows:

````qmd
---
title: "quarto-conditional-content"
format:
    html: default
    pdf: default
---

Some text
[in HTML.]{.content-visible when-format="html"}
[in PDF.]{.content-visible when-format="pdf"}

````

**Comment by cderv** [comment, marked as answer]
It was a regression in Quarto 1.3 - so it was working with 1.2 but not 1.3 and that is solved now in 1.4

* https://github.com/quarto-dev/quarto-cli/issues/6940 (solved by https://github.com/quarto-dev/quarto-cli/pull/7083)

This is why it is documented (as part on 1.2) but not working with 1.3. Current pre-release solves it

**Reply by anielsen001** [reply]
thank you for the quick reply.
